pwiz_tools/Skyline/TestFunctional/AssociateProteinsDlgTest.cs

```diff
@@ -83,7 +88,8 @@ private void TestDialog(ImportType type)
             var dlg2 = ShowDialog<AssociateProteinsDlg>(SkylineWindow.ShowAssociateProteinsDlg);
             if (type == ImportType.FASTA)
             {
-                RunUI(() => dlg2.FastaFileName = _fastaFile);
+                if (Settings.Default.LastProteinAssociationFastaFilepath.IsNullOrEmpty())

```

Comment:
Is this ever not the case anymore? Settings.Default gets reset between test runs.

Reply:
Yes, by doubling up the calls to TestDialog above I am explicitly testing the re-use-last-FASTA-filepath feature.